### ISSUE TITLE
🐛 Bug: Text links aren't distinguished enough from regular text

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [LearningTypeScript/site#53]
### Bug Report Checklist

- [X] I have [searched for related issues](https://github.com/LearningTypeScript/site/issues?q=is%3Aissue+label%3A%22type%3A+bug%22) and found none that matched my issue.
- [X] This is the appropriate issue form for the bug I would like to report.

### Expected

Links should be visible enough against regular text that folks who are low vision / colorblind / etc. can visually distinguish them.

### Actual

There's not enough visual distinction for links compared to regular text now.

### Additional Info

https://usability.yale.edu/web-accessibility/articles/links#designing-links
